docker/package/tezos_setup_wizard.py

```diff
@@ -131,7 +130,7 @@ def is_full_snapshot(import_mode):
     help="Tezos chain offers a Liquidity Baking subsidy mechanism to incentivise exchange\n"
     "between tez and tzBTC. You can ask to end this subsidy, ask to continue it, or abstain.\n"
     "\nYou can read more about this in the here:\n"
-    "https://tezos.gitlab.io/jakarta/liquidity_baking.html",
+    "https://tezos.gitlab.io/kathmandu/liquidity_baking.html",
```

Comment:
let's switch this to:
```suggestion
    "https://tezos.gitlab.io/active/liquidity_baking.html",
```
which will keep working with whatever currently active protocol is running.

IIRC we didn't use this from the beginning because at the time the active protocol didn't have the liquidity option yet, but now it does.